### PR TITLE
Add windows installers for LOM and DDR

### PR DESCRIPTION
**Pull request by fassisrosa**
I believe these will be pre-made installer VS solutions. Small enough to be part of git. 
I tried to create and measure minimal size for build VS solutions but they are just too big even in minimal size.
I will enhance instructions README.md to explain how to create the correct build solutions, where to place them so that installers can pick them up. Good news is that installer uses relative paths so as long as they are placed in same location relative to installer we should be fine.

### CODE REVIEW

File: main/midnight/build.win32/midnight/installer/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Release
```
Comment (fassisrosa) [inline main/midnight/build.win32/midnight/installer/.gitignore:1]: Need to add Debug here as well.

**Comment by fassisrosa** [comment]
Could not find a decent way to have icons be used in build of exe. Do you know how I would do that? Tried variations of configuring proj.win32\game.rc to set icon but no luck. Seems to be ignoring icon.

**Comment by IcemarkUK** [comment]
> Could not find a decent way to have icons be used in build of exe. Do you know how I would do that? Tried variations of configuring proj.win32\game.rc to set icon but no luck. Seems to be ignoring icon.

Did the cocos icon work? Is it that the ico file hasn't generated correctly? Is it only after the install?

**Comment by IcemarkUK** [comment]
Actually this looks familiar - I seem to remember jumping through the hoops when I created the original installers - wish I knew which VM I've left them on!!! :)

**Comment by fassisrosa** [comment]
The installer icons (e.g on desktop, on uninstall list) look fine. The problem is the coco app icon on disk (navigate to program files\chillihugger\midnight and you will see exe file with default cocos icon. Tried changing in game.rc when building but no luck.

**Comment by IcemarkUK** [comment]
ok. I would have expected overwrighting res/game.ico would do it, or editing game.rc - which you've tried - therefore I would be suspicious of game.aps. It looks like compiled .rc file.

**Comment by fassisrosa** [comment]
Updated instructions with detailed steps. You can see how it looks here: https://github.com/fassisrosa/The-Lords-Of-Midnight/tree/feature/beta-1-win

I also merged from latest in your branch and recreated installers. Placed at usual place.

**Comment by IcemarkUK** [comment]
> Updated instructions with detailed steps. You can see how it looks here: https://github.com/fassisrosa/The-Lords-Of-Midnight/tree/feature/beta-1-win
> 
> I also merged from latest in your branch and recreated installers. Placed at usual place.

Looks great!

**Comment by fassisrosa** [comment]
No worries... I'll merge and give it a shot...

BTW: I noticed aspect ratio on Windows changed to widescreen no longer full screen. I like it and it works fine on my screen. But have not tried other display resolutions.

**Comment by IcemarkUK** [comment]
> No worries... I'll merge and give it a shot...
> 
> BTW: I noticed aspect ratio on Windows changed to widescreen no longer full screen. I like it and it works fine on my screen. But have not tried other display resolutions.

This is something I need work out. The original ran Fullscreen or 3 window sizes. You chose from the options menu. And it changed while in the app. However, this might not be so simple in Cocos. At the moment AppDelegate:102 / 147 sets Window or fullscreen. And it's probably the last value I was testing on iPhoneXR

**Comment by fassisrosa** [comment]
Update installer VS solution with new resource locations. All good. Uploaded updated installers to usual place.

**Review by IcemarkUK — APPROVED** [review]

**Pull request merged**